package.json

```diff
@@ -184,5 +184,12 @@
     "supertest": "^7.0.0",
     "ts-jest": "^29.2.4",
     "typescript": "^5.5.4"
+  },
+  "overrides": {
```

Comment:
@GurnankCheema why do you have to force in this REPO locations-inside-prison but not in the other repos else?